src/services/offer.js

```diff
@@ -156,23 +156,40 @@ class OfferService {
         return offer;
     }
 
-    blockByCompany(owner) {
+    hideByCompany(owner, reason) {
```

Comment:
Maybe we should rename to _hideByCompany, in order to hint future devs that they should use the helpers below (blockBy*, unblockBy*, etc...)